kratos/geometries/geometry.h

```diff
@@ -1122,45 +1122,100 @@ class Geometry : public PointerVector<TPointType>
         return false;
     }
 
+    ///@}
+    ///@name Boundaries
+    ///@{
+
+    /**
+     * @brief This method gives you all boundaries entities of this geometry.
+     * @details This method will gives you all the boundaries entities
+     * @return GeometriesArrayType containes this geometry boundaries entities.
+     * @see GeneratePoints()
+     * @see GenerateEdges()
+     * @see GenerateFaces()
+     */
+    virtual GeometriesArrayType GenerateBoundariesEntities( void )
```

Comment:
```suggestion
    virtual GeometriesArrayType GenerateBoundariesEntities const
```

Can we please make this function `const`?

I just figured that the old fcts (e.g. `Edges`, `Faces`) are not `const` and hence I cannot pass the Geometry as `const`

I don't see any reason for them not to be const and since we are anyway updating those functions I think it is a good time to do so

Reply:
I can try, but as the deprecated methods call the new ones..., maybe it will not compile

Reply:
Done

Reply:
Thanks! :tada:

Reply:
shouldn't this be?

virtual GeometriesArrayType GenerateBoundariesEntities() const

with the () ?

Reply:
It is, this comment is outdated (I hate that)